WikiWatershed/model-my-watershed
# Fix project/scenario URL copy

## Overview

ZeroClipboard has become somewhat obsolete since it relies on Flash (which has been turned off by default in Chrome). It has been replaced by Clipboard.js, which operates in a similar way.

Connects #2000 

### Notes

This implementation doesn't work in older version of Safari (as noted [here](https://github.com/zenorocha/clipboard.js#browser-support)). The suggested workaround will require reworking our workflow. In the interest of getting this in for the release, I'm going to create a follow-up issue.

## Testing Instructions

- Load the app, create a project, add a scenario, and save the project. Use the scenario menu to open the share dialog. Make the project public, and press the "Copy Link" button. Verify that the correct URL was added to the clipboard.
- Go through this process in Chrome, Firefox, Safari, and IE.
